doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +262,88 @@ To build and program the source code from the command line, complete the followi
          west flash
 
    The device resets and runs the programmed sample or application.
+
+.. _thingy53_app_update:
+
+Updating firmware image
+=======================
+
+The firmware on Thingy:53 can be programmed using an external debug probe and 10-pin JTAG cable, as described in sections related to building in various environments.
+You can also update applications running on both network and application core using the built-in MCUboot bootloader and nRF Connect Programmer or the nRF Programmer for Android or iOS.
+Prebuilt application images can also be updated that way.
+See subsections below for details about updating firmware image.
+
+Firmware update using external debug probe
+------------------------------------------
+
+If you are using an external debug probe, such as the nRF5340 DK, or any J-Link device supporting ARM Cortex-M33, you do not have to use applications that follow the :ref:`thingy53_app_guide`.
+In that case you can flash the Thingy:53 in a similar way as nRF53 DK, see :ref:`ug_nrf5340` for details.
+Keep in mind that you will need to flash an sample or application compatible with Thingy:53 application guide to bring back serial recovery and DFU OTA.
+
+.. _thingy53_app_update_mcuboot:
+
+Firmware update using MCUboot bootloader
+----------------------------------------
+
+The Thingy:53-compatible samples and applications use MCUboot bootloader than can be used to update firmware.
+This method uses signed binary files :file:`app_update.bin` and :file:`net_core_app_update.bin` (or :file:`dfu_application.zip`).
+You can program the precompiled firmware image using one of the following ways:
+
+* Use the :doc:`mcuboot:index` feature and the built-in serial recovery mode of Thingy:53.
+  In this scenario Thingy is connected directly to your PC through USB.
+* Update the firmware over-the-air (OTA) using Bluetooth Low Energy and the nRF Programmer mobile application for Android or iOS.
```

Comment:
Bluetooth LE? Also, here we again mention nRF Programmer mobile, which would need links to the stores.